terraform/aks/workspace-variables/pen.tfvars.json

```diff
@@ -0,0 +1,40 @@
+{
+  "env_config": "pen",
+  "app_environment": "pen",
+  "key_vault_name": "s189t01-rtt-pen-kv",
+  "key_vault_app_secret_name": "REGISTER-APP-SECRETS-PEN",
+  "key_vault_infra_secret_name": "BAT-INFRA-SECRETS-PEN",
```

Comment:
There are -STAGING secrets in the pen-kv, which I would think should be removed as they are overriding the -PEN secrets. The module loads all secrets from the key vault.

Reply:
removed staging secret and updated the  APPLICATION_HOST .